lib/LMSManagers/LMSFinanceManager.php

```diff
@@ -3761,7 +3761,7 @@ public function GetReceiptList(array $params)
                     $where = ' AND address ?LIKE? ' . $this->db->Escape('%' . $search . '%');
                     break;
                 case 'positions':
-                    $where = ' AND EXISTS (SELECT 1 FROM receiptcontents WHERE description ?LIKE? ' . $this->db->Escape('%' . $search . '%') . ')';
+                    $where = ' AND docid IN (SELECT docid FROM receiptcontents WHERE description ?LIKE? ' . $this->db->Escape('%' . $search . '%') . ')';
```

Comment:
To ogólnie jest ok, ale ciekawe co jest szybsze - czy to co było z dodatkowym warunkiem w WHERE dopasowującym zapytanie wewnętrzne do rekordu z zapytania zewnętrznego czy to co Ty proponujesz.

Reply:
Poprzednie działało źle. Na instancji testowej tego nie wychwyciłem bo miałem 1 rekord w rejestrze kasowym.
Poprzednia wersja działała tak: "Jeśli jest co najmniej jeden rekord w DB z treścią pozycji wyświetl wszystkie wyniki"